hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/WriteStatus.java

```diff
@@ -257,9 +273,19 @@ public boolean isTrackingSuccessfulWrites() {
     return trackSuccessRecords;
   }
 
+  public void setIsMetadata(boolean isMetadataTable) {
+    this.isMetadataTable = isMetadataTable;
+  }
+
+  public boolean isMetadataTable() {
+    return isMetadataTable;
+  }
+
   @Override
   public String toString() {
-    return "WriteStatus {" + "fileId=" + fileId
+    return "WriteStatus {"
+        + "isMetadata=" + isMetadataTable
```

Comment:
`isMetadata` is confusing IMO, actually all the stuff in the write stuff is kind of metadata somehow

Reply:
+1, metadata is an overloaded term. Can we do something like `updatesMetadataTable` to better capture that the write status applies to an update to the metadata table itself